src/Sylius/Bundle/CoreBundle/Migrations/Version20230310160512.php

```diff
@@ -20,21 +20,70 @@ final class Version20230310160512 extends AbstractMigration
 {
     public function getDescription(): string
     {
-        return 'Add ChannelPricingLogEntry';
+        return 'Create ChannelPricingLogEntry, ChannelPriceHistoryConfig and add lowest_price_before_discount, lowest_price_for_discounted_products_checking_period and lowest_price_for_discounted_products_visible columns to sylius_channel table' ;
     }
 
     public function up(Schema $schema): void
     {
+        $this->skipIf(
+            $schema->hasTable('sylius_channel_pricing_log_entry')
+            , 'Skipping migration: Sylius price history migrations were previously executed.'
+        );
+
         $this->addSql('CREATE TABLE sylius_channel_pricing_log_entry (id INT AUTO_INCREMENT NOT NULL, channel_pricing_id INT NOT NULL, price INT NOT NULL, original_price INT DEFAULT NULL, logged_at DATETIME NOT NULL, INDEX IDX_77181A53EADFFE5 (channel_pricing_id), PRIMARY KEY(id)) DEFAULT CHARACTER SET utf8mb4 COLLATE `utf8mb4_unicode_ci` ENGINE = InnoDB');
         $this->addSql('ALTER TABLE sylius_channel_pricing_log_entry ADD CONSTRAINT FK_77181A53EADFFE5 FOREIGN KEY (channel_pricing_id) REFERENCES sylius_channel_pricing (id) ON DELETE CASCADE');
+        $this->addSql('ALTER TABLE sylius_channel_pricing ADD lowest_price_before_discount INT DEFAULT NULL');
+        $this->addSql('ALTER TABLE sylius_channel ADD lowest_price_for_discounted_products_checking_period INT DEFAULT 30 NOT NULL');
+        $this->addSql('CREATE TABLE sylius_channel_excluded_taxons (channel_id INT NOT NULL, taxon_id INT NOT NULL, INDEX IDX_3574E1E972F5A1AA (channel_id), INDEX IDX_3574E1E9DE13F470 (taxon_id), PRIMARY KEY(channel_id, taxon_id)) DEFAULT CHARACTER SET utf8mb4 COLLATE `utf8mb4_unicode_ci` ENGINE = InnoDB');
```

Comment:
I would be in favor of not creating this table, as it is removed a few lines below
https://github.com/Sylius/Sylius/pull/14903/files#diff-129564f0d3ce19ac93be8b110830e662c98c41fa7348c6b76873a203e6117ec8R47